R/iterators.R

```diff
@@ -208,6 +208,17 @@ create_vs <- function(graph, idx, na_ok = FALSE) {
   add_vses_graph_ref(res, graph)
 }
 
+# Internal function to quickly convert integer vectors to igraph.vs
+# for use after c code, when NA and bounds checking is unnecessary
+unsafe_create_vs <- function(graph, idx, verts = NULL){
+  if(is.null(verts))
+    verts <- V(graph)
+
+  res <- simple_vs_index(verts, idx, na_ok = TRUE)
```

Comment:
I guess the reasoning for using `na_ok = TRUE` here is that `simple_vs_index` then does not need to check `any(is.na(res))`? Semantically this is a bit strange. Perhaps the argument should then be renamed `check_na` or something, instead of `na_ok`.

Reply:
`na_ok` is pretty common in other places in `R/iterators.R` and this is all internal anyway, so I'm okay with this.

Semantically, `na_ok` basically means "it's okay if there is an NA value anywhere in the input vector". This _can_ make sense in other invocations of `simple_vs_index`, even though in this particular case we also know that there won't be any NA in the input vector.